backup.config-example

```diff
@@ -27,6 +27,10 @@ GHE_NUM_SNAPSHOTS=10
 #
 #GHE_EXTRA_SSH_OPTS=""
 
+# rsync --bwlimit value. (default no limit)
+#
+#GHE_RSYNC_BWLIMIT=1250
```

Comment:
So to like GHE_EXTRA_SSH_OPTS ?
Try to change as GHE_EXTRA_RSYNC_OPTS.

thanks